flow/src/main/resources/schema/flow/20220713094831_CB-17459_add_jackson_serialized_target_columns.sql

```diff
@@ -0,0 +1,17 @@
+-- // CB-17459 add columns to store Jackson-serialized objects too
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE IF EXISTS flowlog ADD COLUMN IF NOT EXISTS payloadjackson text;
+ALTER TABLE IF EXISTS flowlog ADD COLUMN IF NOT EXISTS variablesjackson text;
+
+ALTER TABLE IF EXISTS flowchainlog ADD COLUMN IF NOT EXISTS triggereventjackson text;
```

Comment:
The entity field `FlowChainLog.triggerEventJackson` is not annotated with `@Column(length = Integer.MAX_VALUE, columnDefinition = "TEXT")`, so either it should be added, or the type here changed to something like `VARCHAR(255)`.

Reply:
It is the same for `triggerEvent` field, working nicely. But for consistency with the `chain` fields, I add the annotation to both places